include/ots-memory-stream.h

```diff
@@ -60,6 +62,8 @@ class ExpandingMemoryStream : public OTSStream {
     return ptr_;
   }
 
+  virtual size_t SizeLimit() { return limit_; }
```

Comment:
```suggestion
  size_t SizeLimit() { return limit_; }
```

Reply:
I was doing the same as WriteRaw which repeats the "virtual" keyword in the derived classes. But I don't like it either, so instead I replaced that with "override" in the derived classes.